lib/beaker-pe/install/pe_utils.rb

```diff
@@ -92,8 +92,8 @@ def install_ca_cert_on(host, opts)
             @cert_cache_dir ||= Dir.mktmpdir("master_ca_cert")
             local_cert_copy = "#{@cert_cache_dir}/ca.pem"
             step "Copying master ca.pem to agent for secure frictionless install" do
-              ca_pem_dir = '/etc/puppetlabs/puppet/ssl/certs'
-              ca_pem_path = "#{ca_pem_dir}/ca.pem"
+              ca_pem_dir = "#{host['puppetpath']}/ssl/certs"
```

Comment:
This looks correct, but perhaps if we change the variable names to:

agent_ca_pem_dir
master_ca_pem_path

Could also change ca_pem_path to be:

ca_pem_path = "#{master['puppetpath']}/ssl/certs/ca.pem"

that might keep us from making the mistake of putting the host[puppetpath] in there?